sshconnect.c

```diff
@@ -500,7 +506,7 @@ ssh_connect_direct(struct ssh *ssh, const char *host, struct addrinfo *aitop,
 				break;
 			} else {
 				oerrno = errno;
-				debug("connect to address %s port %s: %s",
+				debug("connect to %s%s: %s",
 				    ntop, strport, strerror(errno));
```

Comment:
Adapt this error message for unix paths as well.

Reply:
Fixed.